### PR TITLE
Add utility classes for one-liners

### PR DESCRIPTION
**Pull request by lyzadanger**
Part of #2245 

This PR adds more utility classes for one-line styles and cleans up unneeded styles and modules that result.

### CODE REVIEW

File: src/styles/util.scss [hunk 1]
```diff
@@ -1,24 +1,71 @@
...
+}
+
+// Use this class to make an element in a flex container "stretchy" and grow
+// when there is extra space available.
+.u-stretch {
+  flex-grow: 1;
+}
+
+// Put equal, standard spacing between children horizontally
+.u-horizontal-rhythm {
```
Comment (LMS007) [inline src/styles/util.scss:39]: can you add a comment that the spacing is added to the left of each element.

File: src/styles/util.scss [hunk 2]
```diff
@@ -1,24 +1,71 @@
...
+  @include layout.horizontal-rhythm;
+}
+
+// Put equal standard spacing between children vertically
```
Comment (LMS007) [inline src/styles/util.scss:43]: likewise, spacing added to the top

File: src/styles/util.scss [hunk 3]
```diff
@@ -1,24 +1,71 @@
...
+
+// Misc
+
+// Make element content invisible except to assistive technology
 .u-screen-reader-only {
   @include a11y.screen-reader-only;
 }
+
+// Use on elements for a de-emphasized, muted text color
+.u-color-text--muted {
```
Comment (LMS007) [inline src/styles/util.scss:69]: for another pass later on...

should `:disabled` rules have var.$color-text-light ?

File: src/styles/util.scss (hunk 1 above)
Comment (lyzadanger) [reply src/styles/util.scss:39]: Can you elaborate on why you think this is important to comment? Yes, technically the spacing is applied to the left of the element, but is that important to know? The net effect is that there is space between elements. It does not add spacing to the first element, so the net spacing is even. I'm not pushing back so much as trying to understand the need here.

File: src/styles/util.scss (hunk 2 above)
Comment (lyzadanger) [reply src/styles/util.scss:43]: (Same as my previous comment)

File: src/styles/util.scss (hunk 3 above)
Comment (lyzadanger) [reply src/styles/util.scss:69]: Quite possibly! Also, I'm not 100% convinced on the naming of this class, so I'm open to suggestions.

**Pull request merged**